README.md

```diff
@@ -291,6 +291,35 @@ The following table describes types of events.
 | `event.throttle` | The `event.throttle` event emitted  when `librdkafka` reports throttling. |
 | `delivery-report` | The `delivery-report` event is emitted when a delivery report has been found via polling. <br><br>To use this event, you must set `request.required.acks` to `1` or `-1` in topic configuration and `dr_cb` (or `dr_msg_cb` if you want the report to contain the message payload) to `true` in the `Producer` constructor options. |
 
+### Higher Level Producer
+
+The higher level producer is a variant of the producer which can propagate callbacks to you upon message delivery.
+
+```js
+var producer = new Kafka.HighLevelProducer({
+  'metadata.broker.list': 'localhost:9092',
```

Comment:
I'd suggest that we support the `key.serializer` and `value.serializer` config settings that the Java KafkaProducer supports: https://kafka.apache.org/090/javadoc/index.html?org/apache/kafka/clients/producer/KafkaProducer.html

```suggestion
  'metadata.broker.list': 'localhost:9092',
  'key.serializer': NoopSerializer,
  'value.serialzer': NoopSerializer,
```